### DISCUSSION TITLE
Additional CDN server's

### DISCUSSION THREAD

**Discussion by dwisiswant0** [projectdiscovery/cdncheck#7] [Ideas] [answered]
Hi team,

Is this library for WAF CDN server only? I know the original idea is to avoid additional port scans for WAFs, so we don't actually find more results or doesn’t make any sense to run port scans on them. But, I just curious - are Fastly, CloudFront, Azure, and other CDN providers not for that?

**Comment by ehsandeep** [comment]
@dwisiswant0 I believe we can include all the CDN you suggested above, it's up to the user and optional if they want to scan CDN or not.

**Comment by ehsandeep** [comment, marked as answer]
@dwisiswant0 new release is out and we can add CDN / CLOUD / WAF into list now - more details https://github.com/projectdiscovery/cdncheck#how-to-add-new-providers
